egs/wsj/s5/steps/cleanup/internal/get_ctm.sh

```diff
@@ -4,9 +4,9 @@
 
 # This script produces CTM files from a decoding directory that has lattices
 # present.
-# This is similar to get_ctm.sh, but gets the
-# CTM at the utterance-level.
-
+# This is similar to get_ctm.sh, but gets the CTM at the utterance-level.
```

Comment:
This script seems like it's not too closely related to the cleanup scripts and could be moved to steps/get_ctm_fast.sh.  Clarify that it differs from get_ctm.sh because it only uses a single lmwt, operates in parallel over multiple lattices, outputs the ctm to a separate directory, and doesn't  have the option to convert the ctm from an utterance-level ctm to a per-recording ctm.

You should make sure there is a suitable "see also" in get_ctm.sh.